x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/X-ValuesAggregator.java.st

```diff
@@ -94,6 +96,153 @@ $endif$
         return new GroupingState(bigArrays);
     }
 
+$if(BytesRef)$
+    public static GroupingAggregatorFunction.AddInput wrapAddInput(
```

Comment:
Maybe make it a static method somewhere else and reference it? So we don't need to edit it without the IDE.

Reply:
Good idea :)